iosxr_test.py

```diff
@@ -229,19 +237,20 @@ def main():
     global input_box
     global verbose
 
-    parser = argparse.ArgumentParser(description='Pass in a vagrant box')
-    parser.add_argument("a", nargs='?', default="check_string_for_empty")
+    parser = argparse.ArgumentParser(description='Run basic unit-test on a Vagrant VirtualBox')
+    parser.add_argument('BOX_FILE',
+                        help='local Vagrant VirtualBox filename...')
     parser.add_argument('-v', '--verbose',
                         action='store_const', const=logging.DEBUG,
                         default=logging.INFO, help='turn on verbose messages')
 
     args = parser.parse_args()
     verbose = args.verbose
 
-    if args.a == 'check_string_for_empty':
+    if args.BOX_FILE == 'check_string_for_empty':
```

Comment:
This check is now redundant as parse_args() will error out above if you didn't specify a BOX_FILE value.